### DISCUSSION TITLE
大佬们，下载chromego后怎么操作啊

### DISCUSSION THREAD

**Discussion by jiao-art** [bannedbook/fanqiang#841] [Q&A]
我点了第一个clash，然后说我已完成配置，然后怎么用呢

**Comment by bannedbook** [comment]
你有chrome浏览器吗，正常的话会打开一个 chrome窗口

**Reply by REFF417** [reply]
我有chrrome，但是运行程序的时候程序正常运行，但程序不会自动打开chrrome，这是为什么呢？

**Comment by bannedbook** [comment]
重装 chrome 试试
